### PR TITLE
add device arguments for group commands

### PR DESCRIPTION
**Pull request by aequitas**
**Description:**
Title says it all, closes: https://github.com/home-assistant/home-assistant.github.io/issues/2707

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#7577

### CODE REVIEW

File: source/_components/switch.rflink.markdown [hunk 1]
```diff
@@ -52,6 +52,9 @@ Device configuration variables:
 - **aliasses** (*Optional*): Alternative RFLink ID's this device is known by.
 - **fire_event** (*Optional*): Fire an `button_pressed` event if this device is turned on or off (default: False).
 - **signal_repetitions** (*Optional*): Repeat every RFLink command this number of times (default: 1)
+- **group** (*Optional*): Allow switch to respond to group commands (ALLON/ALLOFF). (default: yes)
+- **group_aliasses** (*Optional*): `aliasses` which only respond to group commands.
```
Comment (Landrash) [inline source/_components/switch.rflink.markdown]: Misspelled word aliasses
aliasses -> aliases

File: source/_components/switch.rflink.markdown [hunk 2]
```diff
@@ -52,6 +52,9 @@ Device configuration variables:
...
+- **no_group_aliasses** (*Optional*): `aliasses` which do not respond to group commands.
```
Comment (Landrash) [inline source/_components/switch.rflink.markdown]: Misspelled word aliasses
aliasses -> aliases

File: source/_components/light.rflink.markdown [hunk 3]
```diff
@@ -53,7 +53,9 @@ Device configuration variables:
 - **signal_repetitions** (*Optional*): Repeat every Rflink command this number of times (default: 1).
 - **fire_event_** (*Optional*): Set default `fire_event` for RFLink switch devices (see below).
 - **signal_repetitions** (*Optional*): Set default `signal_repetitions` for RFLink switch devices (see below).
-
+- **group** (*Optional*): Allow light to respond to group commands (ALLON/ALLOFF). (default: yes)
+- **group_aliasses** (*Optional*): `aliasses` which only respond to group commands.
```
Comment (Landrash) [inline source/_components/light.rflink.markdown]: Misspelled word aliasses
aliasses -> aliases

File: source/_components/light.rflink.markdown [hunk 4]
```diff
@@ -53,7 +53,9 @@ Device configuration variables:
...
+- **no_group_aliasses** (*Optional*): `aliasses` which do not respond to group commands.
```
Comment (Landrash) [inline source/_components/light.rflink.markdown]: Misspelled word aliasses
aliasses -> aliases

**Comment by aequitas** [comment]
Corrected spelling, will update sources as well and deprecate incorrect spelled parameters.

**Pull request merged**